### DISCUSSION TITLE
Kivy for Rust

### DISCUSSION THREAD

**Discussion by SanCode1024** [kivy/kivy-github-discussions#24] [General]
Is there plans to create Kivy for Rust programming language. I see that Kivy is great for developing cross-platform (mobile and desktop) GUI based applications. Are there any thoughts on developing Kivy for Rust and utilizing the Kv language for Rust.

**Comment by Julian-O** [comment]
There are no plans, and it seems like an very unlikely step (You are asking a team of Python programmers to redo most of their work in a different language with different strengths, weaknesses and available libraries; it would probably be better off starting again from scratch). Please consult the Kivy Discussions about Road Maps to get involved.

**Comment by alvaAtgithub** [comment]
i was looking for the same thing and found 2 things this  chat and the fact that you can use  PyO3 to interact with python from rust but if your in to rust just use egui eframe combo a bit of serde and jason power you should be able to have a side kivy.py script like .... but its rust just do rust at least that my take away  in adition here are some crates that act like kivy for rust just make a mod on the side to act like kivy.py<Dioxus ,Yew ,OrbTk >
